main.js

```diff
@@ -157,6 +164,7 @@ define(function (require, exports, module) {
         parentWindow.postMessage(JSON.stringify({
             type: "bramble:loaded"
         }), "*");
+        _preferenceSetter()
```

Comment:
Since the function is just running code (without any input or output to/from the function), we don't need it.

Just put the preference toggle here with a comment:

```js
parentWindow.postMessage( 
  //... 
);

// Preferences set here 
PreferencesManager.set(...);
```